### ISSUE TITLE
 Тип подписи

### ISSUE THREAD

**Issue by semeleven** [bankrot/cadesplugin#2]
Приветствую, спасибо большое за труд! У меня такой вопрос - сейчас реализованна подпись только строки? Если да, то будет ли возможность подписать base64 формат?

**Comment by bankrot** [comment]
> сейчас реализованна подпись только строки?

Да, все верно.

> будет ли возможность подписать base64 формат?

Base64 - это ведь тоже строка

**Comment by semeleven** [comment]
Да, верно, просто на офф. сайте есть отдельный метод, который работает с base64 (http://cpdn.cryptopro.ru/default.asp?url=content/cades/plugin-samples-sign-detached.html)

**Comment by dvantage** [comment]
Привет, как можно с тобой связаться?
Уже 2 дня ломаю голову с этим cades, не могу найти метод для выгрузки сертификата открытого ключа (сертификата ключа для проверки ЭП). Чтобы его можно было сразу проверить например на Госуслугах.
